### PR TITLE
Project 14 - Adds Calendar application using Javascript

### PR DESCRIPTION
**Pull request by gyauelvis**
Calendar application
1. Able to add multiple task on the selected date and delete task
2. Show the specific day on the calendar
3. Able to change dates and using the previous and next buttons

### CODE REVIEW

File: .vscode/settings.json
Comment (anishaswain) [inline .vscode/settings.json:1]: Remove this file as it's unnecessary

File: project14[Calendar]/app.js
Comment (anishaswain) [inline project14[Calendar]/app.js:1]: Add another folder inside project14 with your github username (please follow the contribution guidelines mentioned in the readme file)

**Review by anishaswain — CHANGES_REQUESTED** [review]
Please add screen recording of the project as asked in the PR template and change the commit message to something meaningful like "adds calendar application using javascript"

**Comment by gyauelvis** [comment]
> Please add screen recording of the project as asked in the PR template and change the commit message to something meaningful like "adds calendar application using javascript"

I have recorded the video but how do I link the video in the file. Also do I have to host the video on any platform and embed it?I'm confused

**Comment by anishaswain** [comment]
> > Please add screen recording of the project as asked in the PR template and change the commit message to something meaningful like "adds calendar application using javascript"
> 
> I have recorded the video but how do I link the video in the file. Also do I have to host the video on any platform and embed it?I'm confused

Just upload it in the PR description by selecting it from your laptop folder

**Comment by gyauelvis** [comment]
> > > Please add screen recording of the project as asked in the PR template and change the commit message to something meaningful like "adds calendar application using javascript"
> > 
> > I have recorded the video but how do I link the video in the file. Also do I have to host the video on any platform and embed it?I'm confused
> 
> Just upload it in the PR description by selecting it from your laptop folder


https://github.com/anishaswain/Simple-JS-Projects/assets/66832781/150ca3ab-2de1-4fef-8cd8-44fce581f073

**Comment by gyauelvis** [comment]
> Please add screen recording of the project as asked in the PR template and change the commit message to something meaningful like "adds calendar application using javascript"

i have deleted the [.vscode/settings.json](https://github.com/anishaswain/Simple-JS-Projects/pull/43/files/b5f46e6c22cc2c5994e72aa4f7c6f262e1f4748f#diff-a5de3e5871ffcc383a2294845bd3df25d3eeff6c29ad46e3a396577c413bf357)
do I create a new PR?

**Comment by gyauelvis** [comment]
I think I have resolved all the requested changes

File: project14[Calendar]/app.js
Comment (anishaswain) [inline project14[Calendar]/app.js:1]: Add another folder inside project14[Calendar] with your github username (please follow the contribution guidelines mentioned in the readme file)

File: pull_request_template.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ## **Add the working demo (screen recording) below _(Required)_**
 
+
```
Comment (anishaswain) [inline pull_request_template.md:3]: Revert changes to this file

**Review by anishaswain — CHANGES_REQUESTED** [review]

**Comment by anishaswain** [comment]
> I think I have resolved all the requested changes

Added new sets of reviews

**Comment by gyauelvis** [comment]
Done

**Pull request merged**